### ISSUE TITLE
Non-required form slot is not filled from entity when the form is active in Rasa 3.0

### ISSUE THREAD

**Issue by iuliacotaga** [RasaHQ/rasa#10540]
### Rasa Open Source version

3.0.1

### Python version

3.8

### What operating system are you using?

Windows

### What happened?

With the recent migration of model to Rasa Open Source 3.0, I noticed that slot filling within a form did work similarly to Rasa 2.8.

If a slot was defined in domain, but it was not a required slot in a form, then the slot did not get set through entity extraction when the form was active. 

In the attached domain.yml model, `form1_info1` slot was not set with intent `/intent1{"form1_info1":"info1"}`. The entity was correctly identified and extracted, but the mapped slot was not filled in.

In a similar scenario, in Rasa 2.8, ‘form1_info1’ slot was correctly set to the extracted value, although it was not specified as form required slot.

In Rasa 3.0, once `form1_info1` slot was added to the form, it was correctly set with `/intent1{“form1_info1”:“info1”}` user message.

### Command / Request

```shell
Your input ->  /form_start
intent1 will fill slot1 and info1 slots
Your input ->  /intent1{"form1_info1":"info1"}
intent2 will fill slot2 slot
Your input ->  /intent2
form1_slot1 is Filled, form1_slot2 is Filled, form1_info1 is None
```

### Relevant log output

```shell
Your input ->  /form_start
2021-12-10 16:58:01 DEBUG    rasa.core.lock_store  - Issuing ticket for conversation '760dff531743417d993523a150fe2f53'.
2021-12-10 16:58:01 DEBUG    rasa.core.lock_store  - Acquiring lock for conversation '760dff531743417d993523a150fe2f53'.
2021-12-10 16:58:01 DEBUG    rasa.core.lock_store  - Acquired lock for conversation '760dff531743417d993523a150fe2f53'.
2021-12-10 16:58:01 DEBUG    rasa.core.tracker_store  - Could not find tracker for conversation ID '760dff531743417d993523a150fe2f53'.
2021-12-10 16:58:01 DEBUG    rasa.core.processor  - Starting a new session for conversation ID '760dff531743417d993523a150fe2f53'.
2021-12-10 16:58:01 DEBUG    rasa.core.processor  - Policy prediction ended with events '[]'.
2021-12-10 16:58:01 DEBUG    rasa.core.processor  - Action 'action_session_start' ended with events '[<rasa.shared.core.events.SessionStarted object at 0x7fdf465152e0>, ActionExecuted(action: action_listen, policy: None, confidence: None)]'.
2021-12-10 16:58:01 DEBUG    rasa.core.processor  - Current slot values:
        form1_info1: None
        form1_slot1: None
        form1_slot2: None
        requested_slot: None
        session_started_metadata: None
2021-12-10 16:58:01 DEBUG    rasa.engine.runner.dask  - Running graph with inputs: {'__message__': [<rasa.core.channels.channel.UserMessage object at 0x7fdf46517790>]}, targets: ['run_RegexMessageHandler'] and ExecutionContext(model_id='a135329785a9449b91605fc10c21ffa5', should_add_diagnostic_data=False, is_finetuning=False, 
node_name=None).
2021-12-10 16:58:01 DEBUG    rasa.engine.graph  - Node 'nlu_message_converter' running 'NLUMessageConverter.convert_user_message'.
2021-12-10 16:58:01 DEBUG    rasa.engine.graph  - Node 'domain_provider' running 'DomainProvider.provide_inference'.
2021-12-10 16:58:01 DEBUG    rasa.engine.graph  - Node 'run_RegexMessageHandler' running 'RegexMessageHandler.process'.
2021-12-10 16:58:01 DEBUG    rasa.core.processor  - Received user message '/form_start' with intent '{'name': 'form_start', 'confidence': 1.0}' and entities '[]'
2021-12-10 16:58:01 DEBUG    rasa.core.processor  - Logged UserUtterance - tracker now has 4 events.
2021-12-10 16:58:01 DEBUG    rasa.core.actions.action  - Validating extracted slots:
2021-12-10 16:58:01 DEBUG    rasa.core.processor  - Default action 'action_extract_slots' was executed, resulting in 0 events:
2021-12-10 16:58:01 DEBUG    rasa.engine.runner.dask  - Running graph with inputs: {'__tracker__': <rasa.shared.core.trackers.DialogueStateTracker object at 0x7fdf46517820>}, targets: ['select_prediction'] and ExecutionContext(model_id='a135329785a9449b91605fc10c21ffa5', should_add_diagnostic_data=False, is_finetuning=False, 
node_name=None).
2021-12-10 16:58:01 DEBUG    rasa.engine.graph  - Node 'rule_only_data_provider' running 'RuleOnlyDataProvider.provide'.
2021-12-10 16:58:01 DEBUG    rasa.engine.graph  - Node 'domain_provider' running 'DomainProvider.provide_inference'.
2021-12-10 16:58:01 DEBUG    rasa.engine.graph  - Node 'run_MemoizationPolicy0' running 'MemoizationPolicy.predict_action_probabilities'.
2021-12-10 16:58:01 DEBUG    rasa.core.policies.memoization  - Current tracker state:
[state 1] user intent: form_start | previous action name: action_listen
2021-12-10 16:58:01 DEBUG    rasa.core.policies.memoization  - There is no memorised next action
2021-12-10 16:58:01 DEBUG    rasa.engine.graph  - Node 'run_RulePolicy1' running 'RulePolicy.predict_action_probabilities'.
2021-12-10 16:58:01 DEBUG    rasa.core.policies.rule_policy  - Current tracker state:
[state 1] user text: /form_start | previous action name: action_listen
2021-12-10 16:58:01 DEBUG    rasa.core.policies.rule_policy  - There is no applicable rule.
2021-12-10 16:58:01 DEBUG    rasa.core.policies.rule_policy  - Current tracker state:
[state 1] user intent: form_start | previous action name: action_listen
2021-12-10 16:58:01 DEBUG    rasa.core.policies.rule_policy  - There is a rule for the next action 'form1'.  
2021-12-10 16:58:01 DEBUG    rasa.engine.graph  - Node 'select_prediction' running 'DefaultPolicyPredictionEnsemble.combine_predictions_from_kwargs'.
2021-12-10 16:58:01 DEBUG    rasa.core.policies.ensemble  - Made prediction using user intent.
2021-12-10 16:58:01 DEBUG    rasa.core.policies.ensemble  - Added `DefinePrevUserUtteredFeaturization(False)` event.
2021-12-10 16:58:01 DEBUG    rasa.core.policies.ensemble  - Predicted next action using RulePolicy.
2021-12-10 16:58:01 DEBUG    rasa.core.processor  - Predicted next action 'form1' with confidence 1.00.      
2021-12-10 16:58:01 DEBUG    rasa.core.actions.forms  - Activated the form 'form1'.
2021-12-10 16:58:01 DEBUG    rasa.core.actions.forms  - No pre-filled required slots to validate.
2021-12-10 16:58:01 DEBUG    rasa.core.actions.forms  - Request next slot 'form1_slot1'
2021-12-10 16:58:01 DEBUG    rasa.core.processor  - Policy prediction ended with events '[<rasa.shared.core.events.DefinePrevUserUtteredFeaturization object at 0x7fdf464484f0>]'.
2021-12-10 16:58:01 DEBUG    rasa.core.processor  - Action 'form1' ended with events '[<rasa.shared.core.events.ActiveLoop object at 0x7fdf46527640>, <rasa.shared.core.events.SlotSet object at 0x7fdf464482e0>, BotUttered('intent1 will fill slot1 and info1 slots', {"elements": null, "quick_replies": null, "buttons": null, "attachment": null, "image": null, "custom": null}, {"utter_action": "utter_ask_form1_slot1"}, 1639155481.1879516)]'.
2021-12-10 16:58:01 DEBUG    rasa.core.processor  - Current slot values:
        form1_info1: None
        form1_slot1: None
        form1_slot2: None
        requested_slot: form1_slot1
        session_started_metadata: None
2021-12-10 16:58:01 DEBUG    rasa.engine.runner.dask  - Running graph with inputs: {'__tracker__': <rasa.shared.core.trackers.DialogueStateTracker object at 0x7fdf46517820>}, targets: ['select_prediction'] and ExecutionContext(model_id='a135329785a9449b91605fc10c21ffa5', should_add_diagnostic_data=False, is_finetuning=False, 
node_name=None).
2021-12-10 16:58:01 DEBUG    rasa.engine.graph  - Node 'rule_only_data_provider' running 'RuleOnlyDataProvider.provide'.
2021-12-10 16:58:01 DEBUG    rasa.engine.graph  - Node 'domain_provider' running 'DomainProvider.provide_inference'.
2021-12-10 16:58:01 DEBUG    rasa.engine.graph  - Node 'run_MemoizationPolicy0' running 'MemoizationPolicy.predict_action_probabilities'.
2021-12-10 16:58:01 DEBUG    rasa.core.policies.memoization  - Current tracker state:
2021-12-10 16:58:01 DEBUG    rasa.core.policies.memoization  - There is no memorised next action
2021-12-10 16:58:01 DEBUG    rasa.engine.graph  - Node 'run_RulePolicy1' running 'RulePolicy.predict_action_probabilities'.
2021-12-10 16:58:01 DEBUG    rasa.core.policies.rule_policy  - Predicted 'action_listen' after loop 'form1'. 
2021-12-10 16:58:01 DEBUG    rasa.engine.graph  - Node 'select_prediction' running 'DefaultPolicyPredictionEnsemble.combine_predictions_from_kwargs'.
2021-12-10 16:58:01 DEBUG    rasa.core.policies.ensemble  - Predicted next action using RulePolicy.
2021-12-10 16:58:01 DEBUG    rasa.core.processor  - Predicted next action 'action_listen' with confidence 1.00.
2021-12-10 16:58:01 DEBUG    rasa.core.processor  - Policy prediction ended with events '[]'.
2021-12-10 16:58:01 DEBUG    rasa.core.processor  - Action 'action_listen' ended with events '[]'.
2021-12-10 16:58:01 DEBUG    rasa.core.lock_store  - Deleted lock for conversation '760dff531743417d993523a150fe2f53'.
intent1 will fill slot1 and info1 slots
Your input ->  /intent1{"form1_info1":"info1"}
2021-12-10 16:58:06 DEBUG    rasa.core.lock_store  - Issuing ticket for conversation '760dff531743417d993523a150fe2f53'.
2021-12-10 16:58:06 DEBUG    rasa.core.lock_store  - Acquiring lock for conversation '760dff531743417d993523a150fe2f53'.
2021-12-10 16:58:06 DEBUG    rasa.core.lock_store  - Acquired lock for conversation '760dff531743417d993523a150fe2f53'.
2021-12-10 16:58:06 DEBUG    rasa.core.tracker_store  - Recreating tracker for id '760dff531743417d993523a150fe2f53'
2021-12-10 16:58:06 DEBUG    rasa.engine.runner.dask  - Running graph with inputs: {'__message__': [<rasa.core.channels.channel.UserMessage object at 0x7fdf46da2bb0>]}, targets: ['run_RegexMessageHandler'] and ExecutionContext(model_id='a135329785a9449b91605fc10c21ffa5', should_add_diagnostic_data=False, is_finetuning=False, 
node_name=None).
2021-12-10 16:58:06 DEBUG    rasa.engine.graph  - Node 'nlu_message_converter' running 'NLUMessageConverter.convert_user_message'.
2021-12-10 16:58:06 DEBUG    rasa.engine.graph  - Node 'domain_provider' running 'DomainProvider.provide_inference'.
2021-12-10 16:58:06 DEBUG    rasa.engine.graph  - Node 'run_RegexMessageHandler' running 'RegexMessageHandler.process'.
2021-12-10 16:58:06 DEBUG    rasa.core.processor  - Received user message '/intent1{"form1_info1":"info1"}' with intent '{'name': 'intent1', 'confidence': 1.0}' and entities '[{'entity': 'form1_info1', 'value': 'info1', 'start': 8, 'end': 31, 'extractor': 'RegexMessageHandler'}]'
2021-12-10 16:58:06 DEBUG    rasa.core.processor  - Current slot values:
        form1_info1: None
        form1_slot1: None
        form1_slot2: None
        requested_slot: form1_slot1
        session_started_metadata: None
2021-12-10 16:58:06 DEBUG    rasa.core.processor  - Logged UserUtterance - tracker now has 11 events.        
2021-12-10 16:58:06 DEBUG    rasa.core.actions.action  - Validating extracted slots: form1_slot1
2021-12-10 16:58:06 DEBUG    rasa.core.processor  - Default action 'action_extract_slots' was executed, resulting in 1 events: SlotSet(key: form1_slot1, value: Filled)
2021-12-10 16:58:06 DEBUG    rasa.engine.runner.dask  - Running graph with inputs: {'__tracker__': <rasa.shared.core.trackers.DialogueStateTracker object at 0x7fdf46da2dc0>}, targets: ['select_prediction'] and ExecutionContext(model_id='a135329785a9449b91605fc10c21ffa5', should_add_diagnostic_data=False, is_finetuning=False, 
node_name=None).
2021-12-10 16:58:06 DEBUG    rasa.engine.graph  - Node 'rule_only_data_provider' running 'RuleOnlyDataProvider.provide'.
2021-12-10 16:58:06 DEBUG    rasa.engine.graph  - Node 'domain_provider' running 'DomainProvider.provide_inference'.
2021-12-10 16:58:06 DEBUG    rasa.engine.graph  - Node 'run_MemoizationPolicy0' running 'MemoizationPolicy.predict_action_probabilities'.
2021-12-10 16:58:06 DEBUG    rasa.core.policies.memoization  - Current tracker state:
[state 1] user intent: intent1 | user entities: ('form1_info1',) | previous action name: action_listen       
2021-12-10 16:58:06 DEBUG    rasa.core.policies.memoization  - There is no memorised next action
2021-12-10 16:58:06 DEBUG    rasa.engine.graph  - Node 'run_RulePolicy1' running 'RulePolicy.predict_action_probabilities'.
2021-12-10 16:58:06 DEBUG    rasa.core.policies.rule_policy  - Current tracker state:
[state 1] user intent: form_start | previous action name: action_listen
[state 2] user intent: form_start | previous action name: form1 | active loop: {'name': 'form1'}
[state 3] user text: /intent1{"form1_info1":"info1"} | previous action name: action_listen | active loop: {'name': 'form1'}
2021-12-10 16:58:06 DEBUG    rasa.core.policies.rule_policy  - There is no applicable rule.
2021-12-10 16:58:06 DEBUG    rasa.core.policies.rule_policy  - Predicted loop 'form1'.
2021-12-10 16:58:06 DEBUG    rasa.engine.graph  - Node 'select_prediction' running 'DefaultPolicyPredictionEnsemble.combine_predictions_from_kwargs'.
2021-12-10 16:58:06 DEBUG    rasa.core.policies.ensemble  - Made prediction using user intent.
2021-12-10 16:58:06 DEBUG    rasa.core.policies.ensemble  - Added `DefinePrevUserUtteredFeaturization(False)` event.
2021-12-10 16:58:06 DEBUG    rasa.core.policies.ensemble  - Predicted next action using RulePolicy.
2021-12-10 16:58:06 DEBUG    rasa.core.processor  - Predicted next action 'form1' with confidence 1.00.      
2021-12-10 16:58:06 DEBUG    rasa.core.actions.forms  - Validating user input 'UserUttered(text: /intent1{"form1_info1":"info1"}, intent: intent1, entities: info1 (Type: form1_info1, Role: None, Group: None), use_text_for_featurization: False)'.
2021-12-10 16:58:06 DEBUG    rasa.core.actions.forms  - Validating extracted slots: {'form1_slot1': 'Filled'}2021-12-10 16:58:06 DEBUG    rasa.core.actions.forms  - Request next slot 'form1_slot2'
2021-12-10 16:58:06 DEBUG    rasa.core.processor  - Policy prediction ended with events '[<rasa.shared.core.events.DefinePrevUserUtteredFeaturization object at 0x7fdf46d9b400>]'.
2021-12-10 16:58:06 DEBUG    rasa.core.processor  - Action 'form1' ended with events '[<rasa.shared.core.events.SlotSet object at 0x7fdf40b34160>, <rasa.shared.core.events.SlotSet object at 0x7fdf40be1790>, BotUttered('intent2 will fill slot2 slot', {"elements": null, "quick_replies": null, "buttons": null, "attachment": null, "image": null, "custom": null}, {"utter_action": "utter_ask_form1_slot2"}, 1639155486.7616467)]'.
2021-12-10 16:58:06 DEBUG    rasa.core.processor  - Current slot values:
        form1_info1: None
        form1_slot1: Filled
        form1_slot2: None
        requested_slot: form1_slot2
        session_started_metadata: None
2021-12-10 16:58:06 DEBUG    rasa.engine.runner.dask  - Running graph with inputs: {'__tracker__': <rasa.shared.core.trackers.DialogueStateTracker object at 0x7fdf46da2dc0>}, targets: ['select_prediction'] and ExecutionContext(model_id='a135329785a9449b91605fc10c21ffa5', should_add_diagnostic_data=False, is_finetuning=False, 
node_name=None).
2021-12-10 16:58:06 DEBUG    rasa.engine.graph  - Node 'rule_only_data_provider' running 'RuleOnlyDataProvider.provide'.
2021-12-10 16:58:06 DEBUG    rasa.engine.graph  - Node 'domain_provider' running 'DomainProvider.provide_inference'.
2021-12-10 16:58:06 DEBUG    rasa.engine.graph  - Node 'run_MemoizationPolicy0' running 'MemoizationPolicy.predict_action_probabilities'.
2021-12-10 16:58:06 DEBUG    rasa.core.policies.memoization  - Current tracker state:
2021-12-10 16:58:06 DEBUG    rasa.core.policies.memoization  - There is no memorised next action
2021-12-10 16:58:06 DEBUG    rasa.engine.graph  - Node 'run_RulePolicy1' running 'RulePolicy.predict_action_probabilities'.
2021-12-10 16:58:06 DEBUG    rasa.core.policies.rule_policy  - Predicted 'action_listen' after loop 'form1'. 
2021-12-10 16:58:06 DEBUG    rasa.engine.graph  - Node 'select_prediction' running 'DefaultPolicyPredictionEnsemble.combine_predictions_from_kwargs'.
2021-12-10 16:58:06 DEBUG    rasa.core.policies.ensemble  - Predicted next action using RulePolicy.
2021-12-10 16:58:06 DEBUG    rasa.core.processor  - Predicted next action 'action_listen' with confidence 1.00.
2021-12-10 16:58:06 DEBUG    rasa.core.processor  - Policy prediction ended with events '[]'.
2021-12-10 16:58:06 DEBUG    rasa.core.processor  - Action 'action_listen' ended with events '[]'.
2021-12-10 16:58:06 DEBUG    rasa.core.lock_store  - Deleted lock for conversation '760dff531743417d993523a150fe2f53'.
intent2 will fill slot2 slot
Your input ->  /intent2
2021-12-10 16:59:37 DEBUG    rasa.core.lock_store  - Issuing ticket for conversation '760dff531743417d993523a150fe2f53'.
2021-12-10 16:59:37 DEBUG    rasa.core.lock_store  - Acquiring lock for conversation '760dff531743417d993523a150fe2f53'.
2021-12-10 16:59:37 DEBUG    rasa.core.lock_store  - Acquired lock for conversation '760dff531743417d993523a150fe2f53'.
2021-12-10 16:59:37 DEBUG    rasa.core.tracker_store  - Recreating tracker for id '760dff531743417d993523a150fe2f53'
2021-12-10 16:59:37 DEBUG    rasa.engine.runner.dask  - Running graph with inputs: {'__message__': [<rasa.core.channels.channel.UserMessage object at 0x7fdf46cf6a30>]}, targets: ['run_RegexMessageHandler'] and ExecutionContext(model_id='a135329785a9449b91605fc10c21ffa5', should_add_diagnostic_data=False, is_finetuning=False, 
node_name=None).
2021-12-10 16:59:37 DEBUG    rasa.engine.graph  - Node 'nlu_message_converter' running 'NLUMessageConverter.convert_user_message'.
2021-12-10 16:59:37 DEBUG    rasa.engine.graph  - Node 'domain_provider' running 'DomainProvider.provide_inference'.
2021-12-10 16:59:37 DEBUG    rasa.engine.graph  - Node 'run_RegexMessageHandler' running 'RegexMessageHandler.process'.
2021-12-10 16:59:37 DEBUG    rasa.core.processor  - Received user message '/intent2' with intent '{'name': 'intent2', 'confidence': 1.0}' and entities '[]'
2021-12-10 16:59:37 DEBUG    rasa.core.processor  - Logged UserUtterance - tracker now has 19 events.        
2021-12-10 16:59:37 DEBUG    rasa.core.actions.action  - Validating extracted slots: form1_slot2
2021-12-10 16:59:37 DEBUG    rasa.core.processor  - Default action 'action_extract_slots' was executed, resulting in 1 events: SlotSet(key: form1_slot2, value: Filled)
2021-12-10 16:59:37 DEBUG    rasa.engine.runner.dask  - Running graph with inputs: {'__tracker__': <rasa.shared.core.trackers.DialogueStateTracker object at 0x7fdf465830d0>}, targets: ['select_prediction'] and ExecutionContext(model_id='a135329785a9449b91605fc10c21ffa5', should_add_diagnostic_data=False, is_finetuning=False, 
node_name=None).
2021-12-10 16:59:37 DEBUG    rasa.engine.graph  - Node 'rule_only_data_provider' running 'RuleOnlyDataProvider.provide'.
2021-12-10 16:59:37 DEBUG    rasa.engine.graph  - Node 'domain_provider' running 'DomainProvider.provide_inference'.
2021-12-10 16:59:37 DEBUG    rasa.engine.graph  - Node 'run_MemoizationPolicy0' running 'MemoizationPolicy.predict_action_probabilities'.
2021-12-10 16:59:37 DEBUG    rasa.core.policies.memoization  - Current tracker state:
[state 1] user intent: intent2 | previous action name: action_listen
2021-12-10 16:59:37 DEBUG    rasa.core.policies.memoization  - There is no memorised next action
2021-12-10 16:59:37 DEBUG    rasa.engine.graph  - Node 'run_RulePolicy1' running 'RulePolicy.predict_action_probabilities'.
2021-12-10 16:59:37 DEBUG    rasa.core.policies.rule_policy  - Current tracker state:
[state 1] user intent: form_start | previous action name: action_listen
[state 2] user intent: form_start | previous action name: form1 | active loop: {'name': 'form1'}
[state 3] user text: /intent2 | previous action name: action_listen | active loop: {'name': 'form1'}
2021-12-10 16:59:37 DEBUG    rasa.core.policies.rule_policy  - There is no applicable rule.
2021-12-10 16:59:37 DEBUG    rasa.core.policies.rule_policy  - Predicted loop 'form1'.
2021-12-10 16:59:37 DEBUG    rasa.engine.graph  - Node 'select_prediction' running 'DefaultPolicyPredictionEnsemble.combine_predictions_from_kwargs'.
2021-12-10 16:59:37 DEBUG    rasa.core.policies.ensemble  - Made prediction using user intent.
2021-12-10 16:59:37 DEBUG    rasa.core.policies.ensemble  - Added `DefinePrevUserUtteredFeaturization(False)` event.
2021-12-10 16:59:37 DEBUG    rasa.core.policies.ensemble  - Predicted next action using RulePolicy.
2021-12-10 16:59:37 DEBUG    rasa.core.processor  - Predicted next action 'form1' with confidence 1.00.      
2021-12-10 16:59:37 DEBUG    rasa.core.actions.forms  - Validating user input 'UserUttered(text: /intent2, intent: intent2, use_text_for_featurization: False)'.
2021-12-10 16:59:37 DEBUG    rasa.core.actions.forms  - Validating extracted slots: {'form1_slot2': 'Filled'}2021-12-10 16:59:37 DEBUG    rasa.core.actions.forms  - Deactivating the form 'form1'
2021-12-10 16:59:37 DEBUG    rasa.core.processor  - Policy prediction ended with events '[<rasa.shared.core.events.DefinePrevUserUtteredFeaturization object at 0x7fdf40b27850>]'.
2021-12-10 16:59:37 DEBUG    rasa.core.processor  - Action 'form1' ended with events '[<rasa.shared.core.events.SlotSet object at 0x7fdf40bb6040>, <rasa.shared.core.events.SlotSet object at 0x7fdf4657bdc0>, <rasa.shared.core.events.ActiveLoop object at 0x7fdf40b348e0>]'.
2021-12-10 16:59:37 DEBUG    rasa.core.processor  - Current slot values:
        form1_info1: None
        form1_slot1: Filled
        form1_slot2: Filled
        requested_slot: None
        session_started_metadata: None
2021-12-10 16:59:37 DEBUG    rasa.engine.runner.dask  - Running graph with inputs: {'__tracker__': <rasa.shared.core.trackers.DialogueStateTracker object at 0x7fdf465830d0>}, targets: ['select_prediction'] and ExecutionContext(model_id='a135329785a9449b91605fc10c21ffa5', should_add_diagnostic_data=False, is_finetuning=False, 
node_name=None).
2021-12-10 16:59:37 DEBUG    rasa.engine.graph  - Node 'rule_only_data_provider' running 'RuleOnlyDataProvider.provide'.
2021-12-10 16:59:37 DEBUG    rasa.engine.graph  - Node 'domain_provider' running 'DomainProvider.provide_inference'.
2021-12-10 16:59:37 DEBUG    rasa.engine.graph  - Node 'run_MemoizationPolicy0' running 'MemoizationPolicy.predict_action_probabilities'.
2021-12-10 16:59:37 DEBUG    rasa.core.policies.memoization  - Current tracker state:
[state 1] previous action name: action_listen
2021-12-10 16:59:37 DEBUG    rasa.core.policies.memoization  - There is no memorised next action
2021-12-10 16:59:37 DEBUG    rasa.engine.graph  - Node 'run_RulePolicy1' running 'RulePolicy.predict_action_probabilities'.
2021-12-10 16:59:37 DEBUG    rasa.core.policies.rule_policy  - Current tracker state:
[state 1] user intent: form_start | previous action name: action_listen
[state 2] user intent: form_start | previous action name: form1
2021-12-10 16:59:37 DEBUG    rasa.core.policies.rule_policy  - There is a rule for the next action 'utter_slots_values'.
2021-12-10 16:59:37 DEBUG    rasa.engine.graph  - Node 'select_prediction' running 'DefaultPolicyPredictionEnsemble.combine_predictions_from_kwargs'.
2021-12-10 16:59:37 DEBUG    rasa.core.policies.ensemble  - Predicted next action using RulePolicy.
2021-12-10 16:59:37 DEBUG    rasa.core.processor  - Predicted next action 'utter_slots_values' with confidence 1.00.
2021-12-10 16:59:37 DEBUG    rasa.core.processor  - Policy prediction ended with events '[]'.
2021-12-10 16:59:37 DEBUG    rasa.core.processor  - Action 'utter_slots_values' ended with events '[BotUttered('form1_slot1 is Filled, form1_slot2 is Filled, form1_info1 is None', {"elements": null, "quick_replies": null, "buttons": null, "attachment": null, "image": null, "custom": null}, {"utter_action": "utter_slots_values"}, 1639155577.4213212)]'.
2021-12-10 16:59:37 DEBUG    rasa.engine.runner.dask  - Running graph with inputs: {'__tracker__': <rasa.shared.core.trackers.DialogueStateTracker object at 0x7fdf465830d0>}, targets: ['select_prediction'] and ExecutionContext(model_id='a135329785a9449b91605fc10c21ffa5', should_add_diagnostic_data=False, is_finetuning=False, 
node_name=None).
2021-12-10 16:59:37 DEBUG    rasa.engine.graph  - Node 'rule_only_data_provider' running 'RuleOnlyDataProvider.provide'.
2021-12-10 16:59:37 DEBUG    rasa.engine.graph  - Node 'domain_provider' running 'DomainProvider.provide_inference'.
2021-12-10 16:59:37 DEBUG    rasa.engine.graph  - Node 'run_MemoizationPolicy0' running 'MemoizationPolicy.predict_action_probabilities'.
2021-12-10 16:59:37 DEBUG    rasa.core.policies.memoization  - Current tracker state:
[state 1] previous action name: action_listen
2021-12-10 16:59:37 DEBUG    rasa.core.policies.memoization  - There is no memorised next action
2021-12-10 16:59:37 DEBUG    rasa.engine.graph  - Node 'run_RulePolicy1' running 'RulePolicy.predict_action_probabilities'.
2021-12-10 16:59:37 DEBUG    rasa.core.policies.rule_policy  - Current tracker state:
[state 1] user intent: form_start | previous action name: action_listen
[state 2] user intent: form_start | previous action name: form1
[state 3] user intent: form_start | previous action name: utter_slots_values
2021-12-10 16:59:37 DEBUG    rasa.core.policies.rule_policy  - There is a rule for the next action 'action_listen'.
2021-12-10 16:59:37 DEBUG    rasa.engine.graph  - Node 'select_prediction' running 'DefaultPolicyPredictionEnsemble.combine_predictions_from_kwargs'.
2021-12-10 16:59:37 DEBUG    rasa.core.policies.ensemble  - Predicted next action using RulePolicy.
2021-12-10 16:59:37 DEBUG    rasa.core.processor  - Predicted next action 'action_listen' with confidence 1.00.
2021-12-10 16:59:37 DEBUG    rasa.core.processor  - Policy prediction ended with events '[]'.
2021-12-10 16:59:37 DEBUG    rasa.core.processor  - Action 'action_listen' ended with events '[]'.
2021-12-10 16:59:37 DEBUG    rasa.core.lock_store  - Deleted lock for conversation '760dff531743417d993523a150fe2f53'.
form1_slot1 is Filled, form1_slot2 is Filled, form1_info1 is None
```

**Definition of done**
- [ ] Fix bug

**Comment by TyDunn** [comment]
@iuliacotaga Can you please share your domain file?

**Comment by ancalita** [comment]
Closed via #10753
